Description: Fix default config path for virtlockd.init
 Re-using the init script later in the rules file. But as with other
 files the default config directory is /etc/default.
Forwarded: no
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt-1.3.1/src/locking/virtlockd.init.in
===================================================================
--- libvirt-1.3.1.orig/src/locking/virtlockd.init.in	2016-01-12 01:12:06.000000000 +0100
+++ libvirt-1.3.1/src/locking/virtlockd.init.in	2016-02-08 11:33:22.091944203 +0100
@@ -37,7 +37,7 @@ PIDFILE=@localstatedir@/run/$SERVICE.pid
 
 VIRTLOCKD_ARGS=
 
-test -f @sysconfdir@/sysconfig/virtlockd && . @sysconfdir@/sysconfig/virtlockd
+test -f @sysconfdir@/default/virtlockd && . @sysconfdir@/default/virtlockd
 
 RETVAL=0
 
